src/main/java/uk/gov/hmcts/cmc/claimstore/models/otherparty/TheirDetails.java

```diff
@@ -92,12 +101,13 @@ public boolean equals(Object obj) {
         return Objects.equals(this.name, that.name)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1067)